reddit.js

```diff
@@ -1,5 +1,41 @@
 var reddit = require("./data.json")
 
 // Write your solutions below
+// console.log(reddit.data.children.length);
+
+
+// #1
+reddit.data.children.forEach(function(allTitles) {
```

Comment:
Here, we are iterating over each *child* object of ```data.children```, so instead of ```allTitles```, I'd recommend ```child``` or ```post``` as a more appropriate parameter name here.